src/Incorporator.ts

```diff
@@ -32,7 +32,15 @@ export default class Incorporator extends PureComponent<Props, State> {
     const handlers = scope.getSelectorHandlers(this.selector);
     for (const evType of Object.keys(handlers)) {
       const onFoo = `on${evType[0].toUpperCase()}${evType.slice(1)}`;
-      props[onFoo] = (ev: any) => handlers[evType]._n(ev);
+      const oldHandler =
+        typeof props[onFoo] === 'function' ? props[onFoo] : undefined;
+      props[onFoo] = (ev: any) => {
```

Comment:
Can you put the `if` condition outside of this arrow function? In the default case where we use the Cycle.js style without callbacks (the most common case when using cycle/react), we shouldn't wrap the handler function in an unnecessary function.